apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -257,15 +271,49 @@ const AvailabilityPage = ({ profile, plan, eventType, workingHours, previousPage
                     {eventType.title}
                   </h1>
                   {eventType?.description && (
-                    <p className="text-bookinglight mb-2 dark:text-white">
+                    <p className="text-bookinglight mb-3 dark:text-white">
```

Comment:
Found that the current width didn't let a number with 2 digits show correctly, so I fixed it.